### PR TITLE
MC-1703 Update REST Token creation docs

### PR DESCRIPTION
**Pull request by aberatcan**
- Added new section
- Added new screenshot

### CODE REVIEW

File: docs/modules/integrate/pages/clustered-rest.adoc [hunk 1]
```diff
@@ -57,11 +57,20 @@ Send a `POST` request to the `/api/tokens` endpoint.
...
-To use the `/api/tokens` endpoint, do the following. Replace the `<username>` and `<password>` placeholders with a user's username and password.
+To use the `/api/tokens` endpoint for other than SAML and OIDC, do the following. Replace the `<username>` and `<password>` placeholders with a user's username and password.
 
 [source,bash]
 ----
-curl -u {<username>}:{<password>} -d '{"username":"<username>"}' -H "Content-Type: application/json" -X POST http://localhost:8080/api/tokens
+curl -u <username>:<password> -d '{"username":"<username>"}' -H "Content-Type: application/json" -X POST http://localhost:8080/api/tokens
+----
+
+To use the `/api/tokens` endpoint for SAML and OIDC, do the following. Replace the `<username>` placeholders with a user's username, and replace the `<x-csrf-token>` and `<cookie>` with X-CSRF-TOKEN and Cookie headers. You can get the headers from Network tab in Developer Tools when you are logged into MC.
```
Comment (promanenko) [inline docs/modules/integrate/pages/clustered-rest.adoc:67]: I think it makes sense to mention that you can get these headers from ANY request, not just the one on the screen

File: docs/modules/integrate/pages/clustered-rest.adoc [hunk 2]
```diff
@@ -57,11 +57,20 @@ Send a `POST` request to the `/api/tokens` endpoint.
 |Send a `POST` request to the `/api/tokens` endpoint.
 |===
 
-To use the `/api/tokens` endpoint, do the following. Replace the `<username>` and `<password>` placeholders with a user's username and password.
+To use the `/api/tokens` endpoint for other than SAML and OIDC, do the following. Replace the `<username>` and `<password>` placeholders with a user's username and password.
```
Comment (promanenko) [inline docs/modules/integrate/pages/clustered-rest.adoc:60]: smth like `for security providers except SAML and OIDC`

File: docs/modules/integrate/pages/clustered-rest.adoc [hunk 3]
```diff
@@ -57,11 +57,20 @@ Send a `POST` request to the `/api/tokens` endpoint.
...
+----
+
+To use the `/api/tokens` endpoint for SAML and OIDC, do the following. Replace the `<username>` placeholders with a user's username, and replace the `<x-csrf-token>` and `<cookie>` with X-CSRF-TOKEN and Cookie headers. You can get the headers from any request on Network tab in Developer Tools when you are logged into MC.
+
+image:ROOT:ClusteredRestProvideHeaders.png[alt=Clustered Rest Provide Headers, align="center"]
+
+[source,bash]
+----
+curl -d '{"username":"<username>"}' -H "X-CSRF-TOKEN: <x-csrf-token>" -H "Cookie: <cookie>" -H "Content-Type: application/json" -X POST http://localhost:8080/api/tokens
 ----
```
Comment (Serdaro) [inline docs/modules/integrate/pages/clustered-rest.adoc:74]: I'd suggest some edits with following layout:

To use the `/api/tokens` endpoint for security providers except SAML and OIDC, run the following command. Replace the `<username>` and `<password>` placeholders with the credentials of a user logged in to the Management Center.

[source,bash]
----
curl -u <username>:<password> -d '{"username":"<username>"}' -H "Content-Type: application/json" -X POST http://localhost:8080/api/tokens
----

To use the `/api/tokens` endpoint for SAML and OIDC, run the following command.

[source,bash]
----
curl -d '{"username":"<username>"}' -H "X-CSRF-TOKEN: <x-csrf-token>" -H "Cookie: <cookie>" -H "Content-Type: application/json" -X POST http://localhost:8080/api/tokens
----

Here, replace the `<username>` placeholder with the username of a user logged in to the Management Center, and replace the `<x-csrf-token>` and `<cookie>` with X-CSRF-TOKEN and Cookie headers, respectively.
You can get the headers from any request on the *Network* tab in *Developer Tools* when you are logged into Management Center:

image:ROOT:ClusteredRestProvideHeaders.png[alt=Clustered Rest Provide Headers, align="center"]

**Review by Serdaro — COMMENTED** [review]
Minor comment.

File: docs/modules/integrate/pages/clustered-rest.adoc (hunk 3 above)
Comment (aberatcan) [reply docs/modules/integrate/pages/clustered-rest.adoc:71]: Done! 👍

**Review by Serdaro — APPROVED** [review]

**Review by promanenko — APPROVED** [review]

**Comment by promanenko** [comment]
please backport this one to 5.1 too

**Pull request merged**